view-admin/src/components/common/JudgementModal/JudgementModal.tsx

```diff
@@ -57,11 +57,12 @@ const JudgementModal = ({
       const currentValue = newNumbers[currentBox];
       const prevValue = currentBox > 0 ? newNumbers[currentBox - 1] : null;
 
-      if (currentValue === "" && value === "0") return;
-
+      if (currentValue === "" && value === "0")
+        setCurrentBox(Math.min(currentBox + 1, 4));
```

Comment:
```suggestion
        setCurrentBox(nextStep);
```
二個以上同様な処理がある場合は再利用性や可読性を考えて一つの変数とか関数にまとめてあげた方良さ
今回の場合だと↓みたいなのを作る。名前はてきとうだから考えて～
`      const nextStep = Math.min(currentBox + 1, 4);`